app/src/main/res/xml/preferences.xml

```diff
@@ -11,4 +11,11 @@
 
     </PreferenceCategory>
 
+    <PreferenceCategory android:title="@string/section">
+        <Preference
+            android:key="@string/section_key"
+            android:title="@string/choose_section" />
+
+    </PreferenceCategory>
```

Comment:
You can set `intent` directly with `Preference`. Easiest way to do that is to declare action intent:
```xml
        <Preference
            android:key="@string/section_key"
            android:title="@string/choose_section" >
            <intent android:action="com.nmakademija.nmaakademija.OnboardingActivity"/>

        </Preference>
```

And add `intent-filter` to `AndroidManifest`:
```xml
        <activity
            android:name=".OnboardingActivity"
            android:configChanges="orientation|screenSize">
            <intent-filter>
                <action android:name="com.nmakademija.nmaakademija.OnboardingActivity"/>
                <category android:name="android.intent.category.DEFAULT" />
            </intent-filter>
        </activity>
```